docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Manage your Applications using Kyma and Compass Console UI
+type: Tutorial
+---
+
+This tutorial presents the basic flow in which you manually register an external Application's API into Compass and expose it into the Kyma Runtime. In the Kyma Runtime, you then create a lambda that calls the `order-service` API. While going through this tutorial, you will navigate between two UI views:
+
+- Compass UI, where you create connections between Applications, Runtimes, and Scenarios
+- Kyma Console UI, where you manage resources used in your Application, such as services, lambdas, and bindings
+
+## Prerequisites
+
+For simplicity reason, use the available Order Service as the sample external Application for this tutorial. Prepare the following:
+
+- [`order-service`](./assets/order-service.yaml) file that contains the service definition, deployment, and its API
+- [API specification](./assets/order-service-api-spec.yaml) of the `order-service`
+- [Lambda function](./assets/lambda.yaml) that calls `order-service` for orders
```

Comment:
and not : the `order-service` as in the above point? whether with not or without it, let's make it consistent throughout the doc

Reply:
I'll skip "the"